src/js/scan.js

```diff
@@ -21,16 +21,19 @@ import { downloadReport } from "./include/download";
 
   activeOnlySwitch.on("change", function () {
     init(checkerList);
+    $("#runButton").prop("disabled", false);
```

Comment:
I think because we're accessing this element multiple times now, we should probably cache that selector, like we do with `activeOnlySwitch`

Reply:
resolved in https://github.com/10up/phpcompat/pull/9/commits/7c8e92d6aab66fe3b01fdf2c12d04862bfd32b8f